test/test_literate.litcoffee

```diff
@@ -34,6 +32,22 @@ The 3rd parameter here indicates that the incoming source is literate.
 This intentionally includes trailing whitespace in code so it also verifies
 that the way `Markdown` spaces are stripped are not also stripping code.
 
-                assert.equal(errors.length, 2)
+                assert.equal(errors.length, 1)
+
+        'Tab indented markdown' :
```

Comment:
I used current style for this file ( see https://github.com/KrysKruk/coffeelint/blame/e718779ed4098a7a0f259c3cd2afdc868967e316/test/test_literate.litcoffee#L15 ).
BTW +1 for no space in this case. Other tests don't use this style, so I'll prepare a PR for that.